BackendApartmentReservation/BackendApartmentReservation.Tests/Integration/FullChecklistManagerTests.cs

```diff
@@ -53,10 +59,19 @@ public async Task CreateChecklistForEmployee_CreatesFullChecklist()
                     CarNumber = "CAR123"
                 };
 
-                var checklist = await checklistManager.CreateChecklistForEmployee(
+                var livingSpaceInfo = new LivingPlaceReservationInfo
+                {
+                    ApartmentReservationInfo = new ApartmentReservationInfo { ApartmentAddress = "Vilnius"},
+                    HotelReservationInfo = new HotelReservationInfo { HotelAddress = "Kaunas"}
+                };
+
+
+            var checklist = await checklistManager.CreateChecklistForEmployee(
                     employee.Id,
                     flightInfo,
-                    carInfo);
+                    carInfo,
+                    livingSpaceInfo
+                    );

```

Comment:
Well, test file is updated, but not the actual tests to reflect changes ;) Not the very best practice

Reply:
Fixing this